utah-design-system-website/src/react/components/websiteContent/guidelines/DataPrivacyDocumentation.jsx

```diff
@@ -1,34 +1,171 @@
-        and accountable digital environment for individuals engaging with government services online.
+        For more information, please refer to the state of Utah{' '}
+        <ExternalLink href="https://www.utah.gov/support/privacypolicy.html">
+          Privacy Policy
+        </ExternalLink>
+        .
       </p>
-      <p>For more information, please refer to the state of Utah <ExternalLink href="https://www.utah.gov/support/privacypolicy.html">Privacy Policy</ExternalLink>.</p>
 
-      <h2 id="footer-links">Customize footer links</h2>
+      <h2 id="footer-links">Customize Footer Links</h2>
       <p>
-        Note: The Utah Footer offers a way for developers to customize the <NavLink to={`${pageUrls.utahFooter}#section-config-custom-links`}>privacy policy and/or terms of use links</NavLink>.
+        Note: The Utah Footer offers a way for developers to customize the{' '}
+        <NavLink to={`${pageUrls.utahFooter}#section-config-custom-links`}>
+          privacy policy and/or terms of use links
+        </NavLink>
+        .
+      </p>
+
+      <h2 id="form-privacy">
+        Form example
+      </h2>
+      <p>
+        Below is an example of a &quot;Notice of Collection / Data
+        Privacy&quot;, where information about how the form data will be used,
+        how long the data will be retained, and what will happen if the user
+        chooses to opt out of the data collection.
+      </p>
+      <p className="mb-auto">
+        To enhance the user experience the Utah Design System suggests the
+        following:
       </p>
+      <ul>
+        <li>
+          Consistent placement of the data collection link. (At the top of the
+          form)
+        </li>
+        <li>Consistent iconography to enhance trust.</li>
+        <li>
+          Consistent wording for the link so that visitors will gain a quicker
+          understanding of what to expect.
+        </li>
+        <li>
+          Visitors can be presented with the information in a variety of ways: link to another page, popup message, popup modal.
+          We have chosen to present a popup modal in this example.
+        </li>
+      </ul>
+      <hr className="mt-spacing" />
+      <div className="static-example mt-spacing-l">
+        <div className="static-example__component-wrapper">
+          <div style={{ width: '70%' }}>
+            <div className="flex items-center justify-center mb-spacing-l">
+              <span className="flex items-center">
+                <span className="utds-icon-before-policy" aria-hidden="true" />
+                <span className="visually-hidden">policy</span>
+              </span>
+              <a
+                href="#privacy"
+                onClick={(event) => {
+                  event.preventDefault();
+                  event.stopPropagation();
+                  setShowModal(!showModal);
```

Comment:
I'm sure you tested this and it works fine. At first glance it causes a little anxiety because the `showModal` has to be mentally proven to not have a chance of being a stale value. Since the function is recreated on every render, it shouldn't ever be stale. A way to not have this panic attack is to write:
```setShowModal((previousValue) => !previousValue)``` :shrug:

Reply:
Will fix

Reply:
Changed how it works. Now N/A.